dev/bddeb

```diff
@@ -61,7 +61,7 @@ def run_helper(helper, args=None, strip=True):
     return stdout
 
 
-def write_debian_folder(root, templ_data, is_python2):
+def write_debian_folder(root, templ_data, is_python2=False):
     """Create a debian package directory with all rendered template files."""
     print("Creating a debian/ folder in %r" % (root))
     if is_python2:
```

Comment:
Can we just drop this code path entirely?

Reply:
Agreed, I pushed another commit to remove the rest of the python2 verbage/conditionals